content/en/docs/doc-contributor-tools/snippets/README.md

```diff
@@ -0,0 +1,40 @@
+# Snippets for Atom
+
+Snippets are bits of text that get inserted into your editor, to save typing and
+reduce syntax errors. The snippets provided in `snippets.cson` are scoped to
+only work on Markdown files within Atom.
+
+## Installation
+
+Copy the contents of the `snippets.cson` file into your existing
+`~/.atom/snippets.cson`. **Do not replace your existing file.**
+
+You do not need to restart Atom.
+
+## Usage
+
+Have a look through `snippets.cson` and note the titles and `prefix` values of
+the snippets.
+
+A given snippet can be triggered in one of two ways:
+
+- By typing the snippet's `prefix` and pressing the `<TAB>` key
+- By searching for the snippet's title in **Packages / Snippets / Available**
+
+For example, open a Markdown file and type `anote` and press `<TAB>`. A blank
+note is added, with the correct Hugo shortcodes.
+
+A snippet can insert a single line or multiple lines of text. Some snippets
+have placeholder values. To get to the next placeholder, press `<TAB>` again.
+
+Some of the snippets only insert partially-formed Markdown or Hugo syntax.
```

Comment:
Nit: snippets insert partially-formed

Reply:
Why?

Reply:
TL,DR: It's a nit.

Longer version:

It depends on whether you use "only" as an adverb or an adjective. Consider an example where it's an adverb:

    "Only" is only the adverb.

And as an adjective:

    "Only" is the only adjective.

For the sentence in this PR, you could reasonably use "only" however you want. Initially, I recommended removing it because it wasn't clear to me whether you intended "only" as an adverb or adjective, and the sentence still works without it. My default preference is that if it's possible to remove words while preserving meaning, economy of language is preferable.

Does that help? In any case:

/lgtm
/approve